ios/engine/KMEI/KeymanEngine/Classes/Model/KMPKeyboard.swift

```diff
@@ -67,13 +67,16 @@ public class KMPKeyboard
         if let languageName = languageJson["name"],
            let languageId = languageJson["id"] {
               self.languages.append(KMPLanguage(name: languageName, languageId: languageId))
+              break
```

Comment:
This seems an inappropriate place to restrict language selection; we want to collect the data about the languages but only install the first one.